homeassistant/components/camera/__init__.py

```diff
@@ -92,6 +125,47 @@ def update_tokens(time):
             hass.async_add_job(entity.async_update_ha_state())
 
     async_track_time_interval(hass, update_tokens, TOKEN_CHANGE_INTERVAL)
+
+    @asyncio.coroutine
+    def async_handle_camera_service(service):
+        """Handle calls to the camera services."""
+        target_cameras = component.async_extract_from_service(service)
+
+        for camera in target_cameras:
+            try:
+                if service.service == SERVICE_ARM:
+                    yield from camera.async_arm()
+                elif service.service == SERVICE_DISARM:
+                    yield from camera.async_disarm()
+            except AttributeError:
+                pass
```

Comment:
We should not just let this pass. We should also not try to update a camera that didn't had an attribute.